hpx/runtime/actions/basic_action.hpp

```diff
@@ -807,6 +836,23 @@ namespace hpx { namespace serialization
 #define HPX_DEFINE_GET_ACTION_NAME(action)                                    \
     HPX_DEFINE_GET_ACTION_NAME_(action, action)                               \
 /**/
+#if defined(HPX_HAVE_ITTNOTIFY) && !defined(HPX_HAVE_APEX)
```

Comment:
This looks like a leftover from the previous ITT branch, could you rebase this PR onto current master, please?